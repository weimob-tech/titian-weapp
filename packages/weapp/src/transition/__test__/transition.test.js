const { getComponent } = require('../../common/test/getComponent');

describe('test component', () => {
  it('should render success', () => {
    const comp = getComponent('transition');
    expect(comp).toMatchSnapshot();
  });
});
