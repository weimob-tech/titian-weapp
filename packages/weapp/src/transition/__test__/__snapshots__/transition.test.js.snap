// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component should render success 1`] = `<main />`;
